src/embind/emval.js

```diff
@@ -32,7 +32,7 @@ var LibraryEmVal = {
       {value: true},
       {value: false},
     );
-    emval_handles.reserved = emval_handles.allocated.length
+    Object.assign(emval_handles, /** @lends {emval_handles} */ { reserved: emval_handles.allocated.length }),
```

Comment:
Separate change?

Reply:
Actually without this closure compile complains about accessing non-existent fields to a struct.  This is apparently the solution.